src/accounts/AccountPage.tsx

```diff
@@ -0,0 +1,67 @@
+// Libraries
+import React, {FC, useContext, useState} from 'react'
+import {Button, IconFont, Overlay, Page} from '@influxdata/clockface'
+
+// Utils
+import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
+import UserAccountProvider from './context/userAccount'
+import AccountTabContainer from './AccountTabContainer'
+
+import {UserAccountContext} from 'src/accounts/context/userAccount'
+
+import {SwitchAccountOverlay} from 'src/accounts/SwitchAccountOverlay'
+
+const AccountAboutPage: FC = () => {
+  const {userAccounts} = useContext(UserAccountContext)
+  const [isSwitchAccountVisible, setSwitchAccountVisible] = useState(false)
+
+  const activeAcctName =
+    userAccounts && userAccounts.filter(acct => acct.isActive)[0].name
```

Comment:
is there guaranteed to be at least one active account? is it guaranteed to have a name?

Reply:
i believe there is guaranteed to be at least one  account with a name (quartz-mock guarantees this). will confirm with randy

Reply:
confirmed with @grace and @distortia that there is guaranteed to be at least one account (since the user has to be logged in to get to this call); and each account is guaranteed to have a name.  

I will add this note as a comment to the code.